migrate/20250825_create_kubernetes_node_objects.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    DB[:kubernetes_node].import(
```

Comment:
```suggestion
    DB[:kubernetes_node].insert(
```

`import` works, but it just calls `insert` if the second argument is a dataset.